iml-agent/src/action_plugins/stratagem/action_filesync.rs

```diff
@@ -26,7 +26,6 @@ fn do_rsync<'a>(
             let output = Command::new("rsync")
                 .arg("-a")
                 .arg("-t")
-                .arg("-X")
```

Comment:
Don't we WANT xattrs? otherwise we lose striping info, right?

Reply:
Right, but if you copy it to NFS the whole thing fails.  We need to create an enhancement for this to duplicate them somehow on the side like other systems do, or only support them for mountpoints that support xattrs.

Reply:
okay.  We should probably check at some point, but we would want to cache answer locally, so it can wait.